src/buildercore/terraform.py

```diff
@@ -555,6 +562,13 @@ def add_table(table_id, table_options):
             'schema': schema_ref,
         }
 
+        if github_token:
```

Comment:
boolean flags are a mess, needs refactoring but this managed to create tables in the `staging` dataset